cli/pipeline/ps.go

```diff
@@ -41,7 +48,7 @@ func pipelinePs(c *cli.Context) error {
 	}
 	repoID, err := internal.ParseRepo(client, repoIDOrFullName)
 	if err != nil {
-		return err
+		return fmt.Errorf("invalid repo-id or repo-full-name: '%s'", repoIDOrFullName)
```

Comment:
Could you test if the changes from #3830 improve it for you already. They were not released yet.

Reply:
Unfortunately it does not, because I do use the woodpecker-cli from outside a project directory / git directory.

Reply:
You should wrap error from `ParseRepo`, at least.
cli/internal/util.go:122

Does `could not get repository` fall into `invalid repo-id or repo-full-name` category?

Reply:
Is this the right way to do it?
My go skills are at a beginners level if at all.